### PR TITLE
Create & Use eAPD Plantinum AMI

### PR DESCRIPTION
**Pull request by cthulhuplus**
This PR has the Packer code to create a "Platinum AMI" from the Golden Image that we receive from CMS. It splits the current aws.user-data.sh script (in Preview/Test) into a bootstrap script and a "install the application" (aws.user-data.sh) script. It also creates a GitHub Action nightly job to create a new Platinum Image.

### This pull request changes...

- Platinum AMI is created from Golden AMI
- Creates bootstrap.sh to install all pre-requisites for the application
- aws.user-data.sh into an application specific install script
- Creates GitHub Action weekly job to create new Platinum AMI
- Creates GitHub Action nightly job to cleanup old & unused AMIs

### This pull request is ready to merge when...

- [ ] Tests have been updated (and all tests are passing)
- [ ] This code has been reviewed by someone other than the original author
- [ ] The experience passes a basic manual accessibility audit (keyboard nav, screenreader, text scaling) OR an exemption is documented
- [ ] The change has been documented
  - [ ] Associated OpenAPI documentation has been updated
- [ ] Changelog is updated as appropriate

### This feature is done when...

- [ ] Design has approved the experience
- [ ] Product has approved the experience

### CODE REVIEW

**Review by thetif — APPROVED** [review]

**Review by tbolt — COMMENTED** [review]
Seems fine from what I can tell. If there's anything I can do beyond reading the code changes, let me know.

File: bin/preview-deploy/aws.amicleanup.sh [hunk 1]
```diff
@@ -0,0 +1,91 @@
+#!/bin/bash
+################## WIP ##################
```
[the remark this answers is not in the document]
Comment (cthulhuplus) [reply bin/preview-deploy/aws.amicleanup.sh:2]: Yes, this is still very much a Work In Progress and all sorts of stuff will go in and come out before I pull the trigger on this.

**Review by eStar386 — APPROVED** [review]

File: terraform/test/test.tf [hunk 2]
```diff
@@ -1,6 +1,9 @@
 ### Provide variables in non-Git tfvars file
 variable "vpc_id" {}
+<<<<<<< HEAD
+=======
 variable "vpc_cidr_block" {}
```
Comment (thetif) [inline terraform/test/test.tf:5]: looks like you might have a merge error to fix

**Review by thetif — DISMISSED** [review]
everything looks good except that merge code

File: terraform/test/test.tf (hunk 2 above)
Comment (cthulhuplus) [reply terraform/test/test.tf:5]: THANK YOU!

**Review by tbolt — APPROVED** [review]

**Review by thetif — APPROVED** [review]

**Pull request merged**